### PR TITLE
New package: fasm-1.73.16

### PR DESCRIPTION
**Pull request by KAction**

### CODE REVIEW

File: srcpkgs/fasm/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'fasm'
+pkgname=fasm
+version=1.73.16
+revision=1
+archs="i686 i686-musl x86_64 x86_64-musl"
+wrksrc=fasm
+short_desc="Fast assembler for the x86 and x86-64 architectures"
+maintainer="Dmitry Bogatov <KAction@disroot.org>"
+license="BSD-2-Clause"
+homepage="http://flatassembler.net"
```
Comment (Chocimier) [inline srcpkgs/fasm/template]: https

File: srcpkgs/fasm/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	_fasm=./fasm
+	_source='source/Linux/fasm.asm'
+	;;
+(x86_64|x86_64-musl)
+	_fasm=./fasm.x64
+	_source='source/Linux/x64/fasm.asm'
+	;;
+esac
+
+do_build() (
```
[the remark this answers is not in the document]
Comment (KAction) [reply srcpkgs/fasm/template]: Relict of previous attempts. Fixed.

File: srcpkgs/fasm/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+short_desc="Fast assembler for the x86 and x86-64 architectures"
+maintainer="Dmitry Bogatov <KAction@disroot.org>"
+license="BSD-2-Clause"
+homepage="http://flatassembler.net"
+distfiles="https://flatassembler.net/fasm-${version}.tgz"
+checksum=f3d87ab5347c315c48644df989a666390c3e4fb0d40daedd19dea242f8a958c9
+nostrip=yes
+
+case ${XBPS_MACHINE} in
+(i686|i686-musl)
```
[the remark this answers is not in the document]
Comment (KAction) [reply srcpkgs/fasm/template]: Ok.

File: srcpkgs/fasm/template [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+wrksrc=fasm
+short_desc="Fast assembler for the x86 and x86-64 architectures"
+maintainer="Dmitry Bogatov <KAction@disroot.org>"
+license="BSD-2-Clause"
+homepage="http://flatassembler.net"
+distfiles="https://flatassembler.net/fasm-${version}.tgz"
+checksum=f3d87ab5347c315c48644df989a666390c3e4fb0d40daedd19dea242f8a958c9
+nostrip=yes
+
+case ${XBPS_MACHINE} in
```
[the remark this answers is not in the document]
Comment (KAction) [reply srcpkgs/fasm/template]: Ok.

File: srcpkgs/fasm/template [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'fasm'
+pkgname=fasm
+version=1.73.16
+revision=1
+archs="i686 i686-musl x86_64 x86_64-musl"
```
[the remark this answers is not in the document]
Comment (KAction) [reply srcpkgs/fasm/template]: Ok.

File: srcpkgs/fasm/template (hunk 1 above)
Comment (KAction) [reply srcpkgs/fasm/template]: Ok.

**Comment by KAction** [comment]
What now?

**Pull request merged**